src/univariate/continuous/loguniform.jl

```diff
@@ -62,3 +66,14 @@ function quantile(d::LogUniform, p::Real)
     p1,a,b = promote(p, params(d)...) # ensure e.g. quantile(LogUniform(1,2), 1f0)::Float32
     exp(p1 * log(b/a)) * a
 end
+
+function kldivergence(p::LogUniform, q::LogUniform)
+    ap, bp, aq, bq = promote(params(p)..., params(q)...)
+    finite = (0 < aq <= ap < bp <= bq)
+    res = log(log(bq / aq) / log(bp / ap))
+    if finite
+        return res
+    else
+        return typeof(res)(Inf)
+    end
```

Comment:
I guess it's a bit more idiomatic to use `oftype(res, Inf)`. Only a matter of taste but I think the following would be a bit nicer:
```suggestion
    return finite ? res : oftype(res, Inf)
```